docs/clients.md

```diff
@@ -0,0 +1,31 @@
+| [Newsout](https://play.google.com/store/apps/details?id=com.inspiredandroid.newsout)                             | Android                      | Apache-2.0 License   | [GitHub](https://github.com/SimonSchubert/NewsOut)                 |
+| [Readrops](https://f-droid.org/en/packages/com.readrops.app/)                                                    | Android                      | GPL-3.0 License      | [GitHub](https://github.com/readrops/Readrops)                     |
+| [CloudNews](https://apps.apple.com/app/cloudnews-owncloud-news-reader/id683859706)                               | iOS                          | BSD-2-Clause License | [GitHub](https://github.com/owncloud/news-ios-app)                 |
+| [Fiery Feeds](https://apps.apple.com/us/app/fiery-feeds-rss-reader/id1158763303)                                 | iOS                          | -                    | -                                                                  |
+| [News Checker](https://chrome.google.com/webstore/detail/owncloud-news-checker/hnmagnmdnfdhabdlicankfbfhcdgbfhe) | Google Chrome                | -                    | [GitHub](https://github.com/owncloud-archive/news-chrome-notifier) |
+| [own News](http://appworld.blackberry.com/webstore/content/32767887/)                                            | Blackberry                   | -                    | -                                                                  |
+| [FeedSpider](http://www.feedspider.net/)                                                                         | Firefox OS, webOS, LuneOS    | MIT License          | [GitHub](https://github.com/OthelloVentures/feedspider)            |
+| [fastReader](http://www.windowsphone.com/en-us/store/app/fastreader/e55e696d-aa45-4a49-bb1c-a1fc7fdabec1)        | Windows Phone                | -                    | -                                                                  |
+| [py3status](https://github.com/ultrabug/py3status/)                                                              | i3wm                         | BSD-3-Clause License | [GitHub](https://github.com/ultrabug/py3status/)                   |
+| [newsboat](http://newsboat.org/)                                                                                 | Unix Terminal                |                      | [GitHub](https://github.com/newsboat/newsboat)                     |
+| [Newsie](https://open-store.io/app/newsie.martinferretti)                                                        | Ubuntu Touch                 | GPL-3.0 License      | [GitLab](https://gitlab.com/ferrettim/newsie)                      |
+
+## Update Clients
+
+An update client uses the nextcloud API or the OCC CLI to update the feeds in News.
+
+* Python mulithread updater https://github.com/nextcloud/news-updater
```

Comment:
```suggestion
* [Python multithread updater](https://github.com/nextcloud/news-updater)
```

But wasn't there an issue with this updater and the latest releases?

Reply:
Well, as far as I know it still works with 15.4.x but I personally don't use it anymore.

I think we can leave it in there, when the repo gets fully abandoned we can remove it from the list or mark it as deprecated.

Reply:
Ah, alright.

btw. `mul_t_ithread` 🙂

Reply:
now fixed :D